### DISCUSSION TITLE
Missing PartialEq for QVariant

### DISCUSSION THREAD

**Discussion by antroids** [KDAB/cxx-qt#486] [General]
Hi there,

Trying to figure out how to use the library and noticed that `cxx_qt_lib::QHash<cxx_qt_lib::QHashPair_QString_QVariant>` cannot be used as `#[qproperty]` due to missing PartialEq impl for QVariant. The comparison is used in generated setter.
Is it intended or just not implemented yet?

**Comment by ahayzen-kdab** [comment]
Thanks for spotting this! Looks like `PartialEq` was missed from `QVariant`, lets turn this into an issue.

**Reply by ahayzen-kdab** [reply]
Moved into #487

**Reply by ahayzen-kdab** [reply]
BTW 0.5.1 is released now which should contain a fix for this, let us know if you still have issues.

**Comment by ahayzen-kdab** [comment]
Closing this discussion as it should be solved, please open new issues or discussions if there are still problems :-)
